### PR TITLE
Doc 503 - Document Global URL

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds information about how a user can find their global URL and why they should use it rather than the generic one.

### CODE REVIEW

File: docs/api-docs/index.md [hunk 1]
```diff
@@ -7,9 +7,37 @@ Macrometa's GDN service offering is API-first. We create API calls for every tas
 We currently have two ways in which you can interact with the API: the app API reference and the docs API reference.
 The first way of using the REST API is the API reference in the GDN web browser interface (app API). Use the built-in API reference to run various calls and view their input and output.
 
-While you can run your API calls from our docs API reference, the second way of using the REST API is through your favorite IDE, terminal, or other interfaces (docs API). 
+While you can run your API calls from our docs API reference, the second way of using the REST API is through your favorite IDE, terminal, or other interfaces (docs API).
```
Comment (jamesm-macrometa) [inline docs/api-docs/index.md]:
```suggestion
You can run API calls from our docs API reference, or alternatively through your favorite IDE, terminal, or other interfaces (docs API).
```

File: docs/api-docs/index.md [hunk 2]
```diff
@@ -7,9 +7,37 @@ Macrometa's GDN service offering is API-first. We create API calls for every tas
...
+
+## Unique Global URL vs. Generic
+
+If you have a Macrometa Play account, then you can use the generic API server address: `https://api-play.paas.macrometa.io`. This address is used in most of our tutorials and getting started examples in the documentation.
+
+However, for production apps, or any account that is not on the Play tier, you must use your unique global URL.
+
+To find your unique API URL:
+
+1. In your Macrometa account, switch to the geofabric that you want to run API calls against. This is the fabric that contains the collections, streams, or other entities that are the target of your API calls.
```
Comment (jamesm-macrometa) [inline docs/api-docs/index.md]:
```suggestion
1. In your Macrometa account, switch to the GeoFabric that you want to run API calls against. This is the fabric that contains the collections, streams, or other entities that are the target of your API calls.
```

We've been referring to geofabrics as "GeoFabrics," but that decision was made a while ago.  Off the top of my head i'm not sure how (or if) GeoFabrics are referred to in the GUI.

File: docs/api-docs/index.md [hunk 3]
```diff
@@ -7,9 +7,37 @@ Macrometa's GDN service offering is API-first. We create API calls for every tas
...
+2. Click **API Reference**.
+
+   The base URL is shown at the top of the screen.
+
+![API URL](/img/api-url.png)
+
+This is the same as the Global URL shown at the top of every geofabric page in the Macrometa web console with `api-` prepended.
```
Comment (jamesm-macrometa) [inline docs/api-docs/index.md]:
```suggestion
This is the same as the Global URL shown at the top of every GeoFabric page in the Macrometa web console with `api-` prepended.
```

File: docs/streams/index.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ persistent://tenant/geofabric/stream-name
 
 ## Streams and Geofabrics
```
Comment (jamesm-macrometa) [inline docs/streams/index.md]:
```suggestion
## Streams and GeoFabrics
```

File: docs/streams/index.md [hunk 5]
```diff
@@ -46,7 +46,7 @@ persistent://tenant/geofabric/stream-name
...
 
-A geofabric is a geo-fenced grouping within a tenant. A tenant can create multiple geofabrics. For more information about geofabrics, refer to [Geofabrics](../geofabrics.md).
+A geofabric is a geo-fenced grouping within a tenant. A tenant can create multiple geofabrics. For more information about geofabrics, refer to [Geofabrics](../geofabrics/index.md).
```
Comment (jamesm-macrometa) [inline docs/streams/index.md]:
```suggestion
A GeoFabric is a geo-fenced grouping within a tenant. A tenant can create multiple GeoFabrics. For more information about GeoFabrics, refer to [GeoFabrics](../geofabrics/index.md).
```

**Review by jamesm-macrometa — APPROVED** [review]
Looks good!  Main comment is that we should be consistent about using "GeoFabric" or "geofabric"

File: docs/api-docs/index.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/api-docs/index.md]: In the GUI, they are Geo Fabrics. Ugh, drives me nuts.

**Review by elof — COMMENTED** [review]
:shipit:

**Pull request merged**